cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/PredicateFunctionsTest.scala

```diff
@@ -27,124 +27,174 @@ class PredicateFunctionsTest extends DocumentingTest {
+        ("predicate", "A predicate that is tested against all items in the list."))
+      query("""MATCH p = (a)-[*1..3]->(b)
+              #WHERE
+              #  a.name = 'Alice'
+              #  AND b.name = 'Daniel'
+              #  AND all(x IN nodes(p) WHERE x.age > 30)
+              #RETURN p""".stripMargin('#'),
+      ResultAssertions(r => {
           r.toList.length should equal(1)
         })) {
-        p("All nodes in the returned paths will have an `age` property of at least *'30'*.")
+        p("All nodes in the returned paths will have a property `age` with a value larger than `30`.")
         resultTable()
       }
     }
     section("any()", "functions-any") {
-      p("`any()` returns true if the predicate holds for at least one element in the given list." +
-        " `null` is returned if the list is `null` or all of its elements are `null`.")
-      function("any(variable IN list WHERE predicate)", "A Boolean.", ("list", "An expression that returns a list. " +
-        "A single element cannot be explicitly passed as a literal in the cypher statement. " +
-        "However, an implicit conversion will happen for a single elements when passing node properties during cypher execution."),
-      ("variable", "This is the variable that can be used from within the predicate."), ("predicate", "A predicate that is tested against all items in the list."))
-      query(
-        """MATCH (a)
-          |WHERE a.name = 'Eskil'
-          |AND any(x IN a.array WHERE x = 'one')
-          |RETURN a.name, a.array""".stripMargin, ResultAssertions(r => {
+      p("""The function `any()` returns `true` if the predicate holds for at least one element in the given list.
+          #`null` is returned if the list is `null` or all of its elements are `null`.""".stripMargin('#'))
+      function("any(variable IN list WHERE predicate)",
+        "A Boolean.",
+        ("list",
+        """An expression that returns a list.
+          #A single element cannot be explicitly passed as a literal in the cypher statement.
+          #However, an implicit conversion will happen for a single elements when passing node properties during cypher execution.""".stripMargin('#')),
+        ("variable", "This is the variable that can be used from within the predicate."),
+        ("predicate", "A predicate that is tested against all items in the list."))
+      query("""MATCH (a)
+              #WHERE any(color IN a.liked_colors WHERE color = 'yellow')
+              #RETURN a.name, a.liked_colors""".stripMargin('#'),
+      ResultAssertions(r => {
           r.toList.length should equal(1)
           r.columnAs[String]("a.name").toList.head should equal("Eskil")
         })) {
-        p("All nodes in the returned paths have at least one *'one'* value set in the array property named `array`.")
+        p("The nodes returned have a property `liked_colors` that is a list containing at least one value with the value `'yellow'`.")
```

Comment:
```suggestion
        p("The query returns nodes with a list of their properties, where at least one property has the value `'yellow'`.")
```

Reply:
I changed the query to return the node instead. And changed the text a little also.